euphonic/plot.py

```diff
@@ -1,50 +1,22 @@
 from typing import List, Optional, Sequence, Tuple, Union
-import warnings
 
 try:
-    import matplotlib as mpl
```

Comment:
I think the removal of this line might be causing the test to fail, changing `name` in the mocked import. Maybe in `mocked_import` using `if 'matplotlib' in name` instead of `if name == 'matplotlib'` might fix it?

Reply:
That's it! Have tweaked the mocker to function more robustly.